packages/fcl-wc/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @onflow/fcl-wc
 
+## 5.0.0
+
+### Patch Changes
+
+- Updated dependencies [[`3d037e84`](https://github.com/onflow/fcl-js/commit/3d037e8470d56935bbc07ad7177ac40907def660)]:
+  - @onflow/fcl@1.7.0
+
```

Comment:
Before merging this bump *must* be manually discarded so that we do not bump this package to 5.0.0